decidim-core/lib/decidim/component_notification_setting_manifest.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ComponentNotificationSettingManifest
+    include ActiveModel::Model
+    include Decidim::AttributeObject::Model
+
+    attribute :name, Symbol
+    attribute :area, Symbol
+    attribute :default_value, String, default: "1"
```

Comment:
Any reason why not doing it with a Boolean type? Something like:

```suggestion
    attribute :default_value, Boolean, default: true
```